### ISSUE TITLE
Hufnágel Pisti nem jobb ember, mint Mézga Géza

### ISSUE THREAD

**Issue by udvaritibor** [bme-notes/adatb#1]
![image](https://cloud.githubusercontent.com/assets/9115891/26804653/05775ad4-4a49-11e7-9521-86ce15472723.png)

**Comment by martonbognar** [comment]
Igen, ez mas jegyzeteknel is letezo problema, csak a peldakodok lettek bemasolva, es azokba a definiciok/tetelek/bizonyitasok torzse, viszont a cimuk a peldaban szereplo maradt, ezeket javitani kell vagy ki kell torolni.
